lib/winapi/libloaderapi.c

```diff
@@ -0,0 +1,37 @@
+#include <libloaderapi.h>
+#include <winbase.h>
+#include <windef.h>
+#include <winerror.h>
+#include <xboxkrnl/xboxkrnl.h>
+
+#include <assert.h>
+#include <stdlib.h>
+
+HMODULE LoadLibraryExA (LPCSTR lpLibFileName, HANDLE hFile, DWORD dwFlags)
+{
+    assert(hFile == NULL);
+    assert(dwFlags == 0);
+
+    // Always fail with not having found the library
+    SetLastError(ERROR_MOD_NOT_FOUND);
+    return NULL;
+}
+
+HMODULE LoadLibraryA (LPCSTR lpLibFileName)
+{
+    return LoadLibraryExA(lpLibFileName, NULL, 0);
+}
+
+BOOL FreeLibrary (HMODULE hLibModule)
+{
+    assert(hLibModule != NULL);
+
+    // Always claim success when free'ing a library
+    return TRUE;
+}
+
+FARPROC GetProcAddress (HMODULE hModule, LPCSTR lpProcName)
+{
+    // Always fail with not having found the export
+    return NULL;
```

Comment:
This is incomplete.

This should be calling `SetLastError`  but I don't know what value. I have already had trouble finding the value for `LoadLibraryExA` (would be nice if someone can also confirm that).

Reply:
`ERROR_PROC_NOT_FOUND` when the function wasn't found in the module, and `ERROR_MOD_NOT_FOUND` if the HANDLE isn't a proper handle to a module.

`ERROR_MOD_NOT_FOUND` for `LoadLibraryExA` is correct.

Reply:
I fail with `ERROR_PROC_NOT_FOUND` now, but left a comment about `ERROR_MOD_NOT_FOUND`.